keyboards/handwired/bluepill/bluepill70/config.h

```diff
@@ -0,0 +1,30 @@
+#ifndef STM70_CONFIG_H_
+#define STM70_CONFIG_H_
+
+#include "../config.h"
```

Comment:
This line is not actually needed, and should be removed. 

Specifically, the build script automatically includes all of the config.h files, so this is not necessary.